docs/src/pages/guides/globalization/globalization.md

```diff
@@ -0,0 +1,48 @@
+# Globalization
```

Comment:
I would prefer the usual "internationalization and localization".

Reply:
I was wondering about the point. I find "globalization" interesting as it's supposed to mean both and is small enough to make a good title. Alternatively, what about using "localization", as Material-UI already implements the infrastructure (internationalization) and it's up to the users to follow the guide to localize?

Reply:
> what about using "localization"

👍

Reply:
Ok, so we rename the page